app-office/ledger/ledger-3.2.0.ebuild

```diff
@@ -0,0 +1,118 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
+
+inherit check-reqs cmake-utils python-single-r1 python-utils-r1 bash-completion-r1
```

Comment:
You can migrate from `cmake-utils.eclass` to `cmake.eclass`.
Also drop direct inheritance of `python-utils-r1.eclass`. It is not required, since other `python*-r1` eclasses are guaranteed to inherit from it, and this ebuild inherits `python-single-r1.eclass`.
Sort inherited eclasses alphabetically.

Reply:
Done.